express/src/app.ts

```diff
@@ -13,9 +13,23 @@ import manageAccount from "./routes/manage-account";
 import signIn from "./routes/sign-in";
 import testingRoutes from "./routes/testing-routes";
 import SelfServiceServicesService from "./services/self-service-services-service";
+import session from "express-session";
+import connect_dynamodb from "connect-dynamodb";
+import AWS from "aws-sdk";
 
 const app = express();
 
+const dynamoDBSessionOptions = {
+    table: process.env.SESSIONS_TABLE,
+    AWSRegion: "eu-west-2",
+    client: process.env.SESSION_LOCAL === "true" ? new AWS.DynamoDB({endpoint: new AWS.Endpoint("http://localhost:8000")}) : ""
```

Comment:
Can we wrap the code here in a function of some sort?  I think it would look neater and explain what we're doing slightly better.

Also, can you consider using an env var like `DYNAMO_DB_ENDPOINT`?  I'm pretty sure you can use a JavaScript-ism like
```
if ( process.env.DYNAMO_DB_ENDPOINT )
   { // return a client with the specified endpoint }
else
   { // return the default client }
```

Then it becomes optional with a sensible default and we have the option to specify a different endpoint (although I can't think why we'd want to right now).  Relying on `'true'` being correctly spelled etc. can be tricky.

Reply:
(Probably put the function in the lib folder somewhere and import it)